Unable to build termite on windows

#### Description
Unable to build Termite on windows.

- pty needs to be bumped to v1.1.18
- `syscall.Kill` doesn't exist on windows

#### Reproduce
```
go build -ldflags="-s -w " -trimpath -o ./build/termite/termite cmd/termite/main.go   
# command-line-arguments
cmd\termite\main.go:113:11: cannot use uint16(msg.Body.(*message.BodyWindowSize).Columns) (value of type uint16) as type uint in struct literal
cmd\termite\main.go:114:11: cannot use uint16(msg.Body.(*message.BodyWindowSize).Rows) (value of type uint16) as type uint in struct literal
cmd\termite\main.go:142:11: cannot use uint16(bodyStartProcess.WindowRows) (value of type uint16) as type uint in struct literal
cmd\termite\main.go:143:11: cannot use uint16(bodyStartProcess.WindowColumns) (value of type uint16) as type uint in struct literal
cmd\termite\main.go:147:19: undefined: pty.StartWithSize
cmd\termite\main.go:298:13: undefined: syscall.Kill
```

#### Environments
 - OS: Windows

--- a/cmd/termite/main.go
+++ b/cmd/termite/main.go
@@ -295,8 +295,12 @@ func handleConnection(c *client) {
 			key := msg.Body.(*message.BodyTerminateProcess).Key
 			log.Success("Request terminate %s", key)
 			if termiteProcess, exists := processes[key]; exists {
-				syscall.Kill(termiteProcess.process.Process.Pid, syscall.SIGTERM)
-				termiteProcess.ptmx.Close()
+				if p, err := os.FindProcess(termiteProcess.process.Process.Pid); err != nil {
+					log.Error("Unable to find process: %s", err)
+				} else {
+					p.Signal(syscall.SIGTERM)
+					termiteProcess.ptmx.Close()
+				}
 			}
 		case message.PULL_TUNNEL_CONNECT:
 			address := msg.Body.(*message.BodyPullTunnelConnect).Address

--- a/go.mod
+++ b/go.mod
@@ -7,7 +7,7 @@ require (
 	github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
 	github.com/asaskevich/govalidator v0.0.0-20200108200545-475eaeb16496 // indirect
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/creack/pty v1.1.14
+	github.com/creack/pty v1.1.18
 	github.com/dustin/go-humanize v1.0.0
 	github.com/elazarl/go-bindata-assetfs v1.0.1
 	github.com/fatih/color v1.12.0

